lib/fireworq.go

```diff
@@ -85,6 +142,23 @@ func (p FireworqPlugin) FetchMetrics() (map[string]float64, error) {
 	m["active_nodes"] = float64(sum.ActiveNodes)
 	m["active_nodes_percentage"] = float64(sum.ActiveNodes*100) / float64(len(stats))
 
+	for q, s := range stats {
+		if s.ActiveNodes >= 1 {
+			// Escape queue name to confirm to metric name specification.
+			// See also: https://mackerel.io/ja/api-docs/entry/host-metrics#post-graphdef
+			r := regexp.MustCompile("[^-a-zA-Z0-9_]")
```

Comment:
This should be put outside the loop or in `init` function since we need it to be initialized once.

Reply:
✅ https://github.com/fireworq/mackerel-plugin-fireworq/pull/1/commits/473d73b0441bdc3da170dc3653ede1e3f73d6f7f